DataType/EntityExistenceDataType.php

```diff
@@ -1,18 +0,0 @@
-<?php
-
-namespace Netdudes\DataSourceryBundle\DataType;
-
-use Netdudes\DataSourceryBundle\Query\FilterCondition;
-
-class EntityExistenceDataType extends EntityDataType
```

Comment:
Is not used.
Is not supported even: [DataSourceBuilder](https://github.com/netdudes/DataSourceryBundle/blob/master/DataSource/DataSourceBuilder.php#L164)